extrinsic_matrix:
- - 0.01990722716241089
  - 0.9996595950708556
  - 0.016864053174640792
  - 0.006557890333006845
- - -0.0028684118835384242
  - 0.016924431662355577
  - -0.999852657058115
  - -0.07213402834271396
- - -0.9997977168007389
  - 0.019855920922465353
  - 0.003204353967691531
  - -0.07457074847358422
- - 0.0
  - 0.0
  - 0.0
  - 1.0
